hack/dump/README.md

```diff
@@ -0,0 +1,140 @@
+# Aggregated Logging Utilities
+
+ ## logging-dump
+`logging-dump.sh` is a tool to gather as much information as possible from your logging cluster. In order to run the script, the user must be logged in and have a administrative permissions on project `logging`
```

Comment:
technically the project name is configurable so there should be an option to supply an alternate name